cypress/integration/marc-authority/marc-authority-view.spec.js

```diff
@@ -0,0 +1,44 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import Features from '../../support/dictionary/features';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import DataImport from '../../support/fragments/data_import/dataImport';
+
+describe('MARC Authority management', () => {
+  const userData = {
+    uniqueFileName: `autotestFile.${getRandomPostfix()}.mrc`
+  };
+
+  before('Creating user', () => {
+    cy.createTempUser([
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorView.gui,
+      Permissions.inventoryAll.gui,
+    ]).then(createdUserProperties => {
+      userData.id = createdUserProperties.userId;
+      userData.firstName = createdUserProperties.firstName;
+      userData.name = createdUserProperties.username;
+      userData.password = createdUserProperties.password;
+
+      cy.loginAsAdmin();
+      DataImport.uploadMarcBib();
```

Comment:
Do you remove it after the test execution?

Reply:
No, I did not find a similar method in similar tests. It will need to be created in the future to remove job and instance. Unfortunately, this was not taken into account in the initial assessment of the task.

Reply:
If we create something we need to remove it. Please don't close the ticket until you create the method to remove the objects you created during the test. This test can influence the results of dataImport autotests.
So you can continue working on this test despite not taking into account this case.